server/TelegramServerConfig.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef TELEGRAMSERVER_CONFIG_HPP
```

Comment:
Please use `TELEGRAM_SERVER_CONFIG_HPP`.
Yes, this stuff is inconsistent yet :-(.